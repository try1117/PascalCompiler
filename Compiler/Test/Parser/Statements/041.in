program test;
const
  a: array[1..5] of integer = (1, 2, 3, 4, 5);
  b: integer = a[2] * a[5];
begin
  write(b + a[1]);
end.
