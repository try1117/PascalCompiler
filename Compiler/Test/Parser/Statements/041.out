test : function()
   resultType : Nil

test declarations:
   a : Const Array [1, 5] of Integer
   |-- Array
   |       |-- 1
   |       |-- 2
   |       |-- 3
   |       |-- 4
   |       --- 5

   b : Const Integer
   |-- 10

|-- Statements
|            |-- Write
|            |       |-- 11
